src/Aquifer.Data/Migrations/20240927213459_bib-746-improve-performance-of-cms-resource-page-2.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Aquifer.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class bib746improveperformanceofcmsresourcepage2 : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_ProjectResourceContents_Projects_ProjectId",
+                table: "ProjectResourceContents");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_ProjectResourceContents_ResourceContents_ResourceContentId",
+                table: "ProjectResourceContents");
+
+            migrationBuilder.AddForeignKey(
+                name: "FK_ProjectResourceContents_Projects_ProjectId",
+                table: "ProjectResourceContents",
+                column: "ProjectId",
+                principalTable: "Projects",
+                principalColumn: "Id",
+                onDelete: ReferentialAction.Cascade);
```

Comment:
I can't get this to use `Restrict` despite setting it [here](https://github.com/BiblioNexusStudio/aquifer-server/pull/457/files#diff-45aaa9b65285c1ecbdf129057bebcea001d00bb135a74fe05fb6bd418db4eda1R25) in the builder.  Any ideas?

Without `Restrict` the migration will fail with:
```
Microsoft.Data.SqlClient.SqlException (0x80131904): Introducing FOREIGN KEY constraint 'FK_ProjectResourceContents_ResourceContents_ResourceContentId' on table 'ProjectResourceContents' may cause cycles or multiple cascade paths. Specify ON DELETE NO ACTION or ON UPDATE NO ACTION, or modify other FOREIGN KEY constraints.
```

Reply:
It looks like the attribute is missing on the class that specifies which configuration to use, e.g. https://github.com/BiblioNexusStudio/aquifer-server/blob/master/src/Aquifer.Data/Entities/UserEntity.cs#L8

Reply:
Thanks, that was part of the problem.  I was also missing an explicit `HasMany()` property.  I updated the PR and now there is no migration code (as desired).